### PR TITLE
Fixes for bleeding-edge GVR builds

### PR DESCRIPTION
**Pull request by ben-clayton**
This is a highly risky, but critical bunch of fixes for GVR.

Please review thoroughly - I'm tired and likely to have made mistakes.

### CODE REVIEW

File: gapis/api/gles/api/extensions.api [hunk 1]
```diff
@@ -642,9 +645,12 @@ cmd void glEGLImageTargetTexture2DOES(GLenum target, GLeglImageOES image) {
...
   }
   t := GetBoundTextureOrErrorInvalidEnum(target)
-  SetTextureImage(t, 0, 0, eglImage.Image)
-  t.EGLImage = eglImage
-  GetEGLImageData(eglImage.ID, eglImage.Image.Width, eglImage.Image.Height)
+
+  for _, l, img in eglImage.Images {
+    SetTextureImage(t, 0, as!GLint(l), img)
+    t.EGLImage = eglImage
+    GetEGLImageData(eglImage.ID, img.Width, img.Height)
```
Comment (dsrbecky) [inline gapis/api/gles/api/extensions.api]: You probably don't want to call this n times...

File: gapis/api/gles/gles.api [hunk 2]
```diff
@@ -119,11 +120,11 @@ class SupportedExtensions {
   bool GL_EXT_primitive_bounding_box                   = true
   bool GL_EXT_raster_multisample                       = true
   bool GL_EXT_robustness                               = true
-  bool GL_EXT_sRGB_write_control                       = true
```
Comment (dsrbecky) [inline gapis/api/gles/gles.api]: It does not matter now, just be aware that once we start using MEC, this class (and most state) will be append-only.

File: gapis/api/gles/api/extensions.api (hunk 1 above)
Comment (ben-clayton) [reply gapis/api/gles/api/extensions.api]: Added a `if l == 0` block around these two statements (we still need the image width / height from somewhere)

File: gapis/api/gles/gles.api (hunk 2 above)
Comment (ben-clayton) [reply gapis/api/gles/gles.api]: Ack

**Pull request merged**